packages/react-components/react-charts-preview/src/components/DonutChart/DonutChartRTL.test.tsx

```diff
@@ -15,6 +15,21 @@ function sharedBeforeEach() {
   resetIds();
 }
 
+beforeAll(() => {
```

Comment:
What is this doing ?

Reply:
ResizeObserver is not defined on window [#3368](https://github.com/jsdom/jsdom/issues/3368). Using this solution i'm not getting this attached error while running the testcases. 
![Screenshot 2024-09-02 171812](https://github.com/user-attachments/assets/751f84b3-4604-4526-aa71-97cd27943b1b)

Reply:
Okay.